default_app/preload.js

```diff
@@ -0,0 +1,13 @@
+const {ipcRenderer, webFrame} = require('electron');
```

Comment:
>  I actually just realized that the zoom level API's being on web-contents is completely undocumented. Is this intended?

Not sure, but it would seem worth making these public